src/Modal/index.jsx

```diff
@@ -14,58 +16,42 @@ class Modal extends React.Component {
     super(props);
 
     this.close = this.close.bind(this);
-    this.handleKeyDown = this.handleKeyDown.bind(this);
-    this.setFirstFocusableElement = this.setFirstFocusableElement.bind(this);
-    this.setCloseButton = this.setCloseButton.bind(this);
 
-    this.closeModalButtonId = newId('paragonCloseModalButton');
     this.headerId = newId();
-    this.el = document.createElement('div');
+    this.modalBodyRef = React.createRef();
 
-    // Sets true for IE11, false otherwise: https://stackoverflow.com/a/22082397/6620612
-    this.isIE11 = !!global.MSInputMethodContext && !!document.documentMode;
+    if (typeof document !== 'undefined') {
```

Comment:
we no longer reference `document` until the component has been hydrated by Gatsby (avoids build time errors).